saleor/graphql/order/bulk_mutations/order_bulk_create.py

```diff
@@ -79,36 +96,35 @@ class OrderBulkOrderLine:
 
 
 @dataclass
-class OrderWithErrors:
+class OrderBulkClass:
```

Comment:
Maybe rename it to `OrderBulkCreateData`? Should be clear name suggesting that this is only a data class gathering different fields.